src/components/Modal/style.scss

```diff
@@ -70,3 +70,13 @@
 .content {
   @extend %layout--bounds;
 }
+
+.pageActions {
```

Comment:
this style was being used in the `Modal` component but wasn't defined. I pulled this from the `Page` component.